source/uwp/NuGet/AdaptiveCards.ObjectModel.Uwp.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="CheckNonUwpApp" Condition="'$(TargetPlatformIdentifier)' != 'UAP'" BeforeTargets="PrepareForBuild">
+        <Error Text="Only UWP apps/libraries are supported." />
+    </Target>
+    <ItemGroup Condition="'$(ProjectExt)' != '.vcxproj'">
+        <SDKReference Include="Microsoft.VCLibs, Version=14.0">
```

Comment:
I may be cautious about this change. instead, I may warn user with error if possible. 

The reason for it is that For C# project, it may depend on Microsoft.VCLibs.140.00.Debug or than Microsoft.VCLibs

Reply:
good callout here -- we should keep an eye out for folks running in to issues, but probably don't have cycles to investigate the possibility at the moment...